### ISSUE TITLE
Have anybody trained Hiera with CLIP?

### ISSUE THREAD

**Issue by MonolithFoundation** [facebookresearch/hiera#38]
Have anybody trained Hiera with CLIP? Will it better than CLIP with Vit?

**Comment by MH-Python** [comment]
I am still learning about Hiera architecture and pre-training. However, I assume in your question you want to pre-train hiera using joint information from language decoder beside the visual decorder, right?
I have never tried to modify the base-line architecture but that idea has been used frequently in other VFMs.

**Comment by MonolithFoundation** [comment]
Yes, this question was posed a significant amount of time ago. I have developed a customized pipeline trained with Hiera in conjunction with CLIP (not actually with a Contrastive learning loss). The outcome is highly satisfactory. It has learned extensively from data, encompassing OCR capabilities for different languages. Thanks to the dynamic input support design of Hiera, it can handle up to 2k input. 


If anyone is interested, they can contact me for the model and codes. Currently, it is not open source.
